### DISCUSSION TITLE
Playing audio via api

### DISCUSSION THREAD

**Discussion by savage7** [koush/scrypted#829] [General]
Does scrypted support an api to send audio to a camera like go2rtc:
POST http://localhost:1984/api/streams?dst=camera1&src=ffmpeg:http://example.com/song.mp3#audio=pcma#input=file

I've searched the docs but didn't find anything.

**Comment by koush** [comment]
Yes, there is an API to play stuff. But it's not a REST API, though one could write a REST using the plugin API.

https://developer.scrypted.app

**Reply by savage7** [reply]
@koush 
Could you guide me a bit how this could work? My goal would be to send audio to a doorbell. It this even possible?

If yes, which classes i.e. Mediaplayer would be involved? I had a look a the dev documentation and looked at some plugins, but i did find something similar.   

This looked promising https://github.com/koush/scrypted-google-cloud-tts but what a don't get is that this only converts. How will this be triggered and how does it send the audio to let's say a two way audio camera?
